pageserver/src/page_service.rs

```diff
@@ -490,17 +484,28 @@ impl PageServerHandler {
                 }
             };
 
+            if let Err(e) = &response {
+                if timeline.cancel.is_cancelled() {
+                    // If we fail to fulfil a request during shutdown, which may be _because_ of
+                    // shutdown, then do not send the error to the client.  Instead just drop the
+                    // connection.
+                    span.in_scope(|| info!("dropped request during shutdown: {e}"));
```

Comment:
```suggestion
                    span.in_scope(|| info!("dropped response during shutdown: {e:#}"));
```

- request -> response
- formatting (the block below uses display alternate so this is probably an anyhow)